src/Common/tests/System/Net/Prerequisites/Servers/CoreFxNetCloudService/WebServer/WebServer.csproj

```diff
@@ -79,25 +79,25 @@
       <HintPath>..\packages\Newtonsoft.Json.5.0.8\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="System" />
+    <Reference Include="System.ComponentModel.DataAnnotations" />
+    <Reference Include="System.Configuration" />
+    <Reference Include="System.Data" />
     <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="System.Drawing" />
+    <Reference Include="System.EnterpriseServices" />
     <Reference Include="System.Net.Http" />
     <Reference Include="System.Spatial, Version=5.6.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\packages\System.Spatial.5.6.2\lib\net40\System.Spatial.dll</HintPath>
```

Comment:
>  If this shouldn't be built at all

This should and is built whenever we deploy to the Azure machines or deploy the internal System.Net multi-machine environment. Please follow the readme.md process to do so. Given that multi-machine testing is not yet possible, I haven't enabled this part of the normal CoreFX builds.

@joperezr Please point me to the public repo hosting the Dev/eng world. Is there a reason it can't follow the readme.md to build and deploy this as required for System.Net Outerloop testing in a multi-machine environment?